ganghee/MyStudy2/app/src/main/java/com/example/mystudy/di/NetworkModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.mystudy.di
+
+import com.example.mystudy.data.remote.UpbitService
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+fun getRemoteServiceModules(url: String) = module {
```

Comment:
```suggestion
val remoteServiceModule = module {
```
아래 리뷰에따라 url을 인자로 따로 받는것이 어떨까 합니다.

Reply:
위의 리뷰가 이유입니다!